lib/github.sh

```diff
@@ -205,6 +203,21 @@ strap::github::api::token::create() {
     strap::abort "$msg"
   fi
 
+  #Test if we need to do an SSO enable
+  response="$(curl --silent --show-error -i -H "Authorization: token $token" -H 'Content-Type: application/json' https://api.github.com/user/issues)"
+  headers="$(echo "$response" | sed "/^\s*$(printf '\r')*$/q" | sed '/^[[:space:]]*$/d' | tail -n +2)"
+  if echo "$headers" | grep -q 'X-GitHub-SSO:'; then
+    # Linux based system
+    if command -v gio >/dev/null; then
+      gio open https://github.com/settings/tokens
+    # Mac
+    else
+      open https://github.com/settings/tokens
+    fi
+    echo "One or more of your Github Org requires you to enable the token before using. See more details on https://help.github.com/en/articles/authorizing-a-personal-access-token-for-use-with-a-saml-single-sign-on-organization."
+    read -r -p "Press [Enter] key once you have enabled the token..."  ignore </dev/tty
```

Comment:
doing `read -p ".."` seems to break the script.

Reply:
It works for me in zsh on macOS? What platform / shell are you seeing it break on @brianhuang-okta ?

Reply:
I just testing this on a CentOS VM with bash and it worked fine.

Reply:
Can you show me a screenshot of running strap on CentOS albatrap to show this is true? I found just read -p would end the strap run prematurely.

Reply:
`-r -p` is sufficient - it is what strap uses in it's primary `readval` function, and is already in use in both Mac and CentOS when prompting the user for other information.

https://github.com/ultimatedotfiles/strap/blob/master/lib/io.sh#L61

Reply:
it needs to be written to a variable otherwise the behaviour is unexpected. WIthout writing it to ignore it exits prematurely. 
Without writing to var
<img width="789" alt="Screen Shot 2019-08-22 at 4 41 16 PM" src="https://user-images.githubusercontent.com/28074204/63548438-387c8c80-c4fc-11e9-9f8c-3fd28e68421b.png">
Writing to an ignore var
<img width="791" alt="Screen Shot 2019-08-22 at 4 44 31 PM" src="https://user-images.githubusercontent.com/28074204/63548439-387c8c80-c4fc-11e9-88e9-82f851f794d6.png">